engine/src/flutter/lib/ui/semantics/semantics_node.h

```diff
@@ -163,13 +154,12 @@ struct SemanticsNode {
   ~SemanticsNode();
 
   bool HasAction(SemanticsAction action) const;
-  bool HasFlag(SemanticsFlags flag) const;
 
   // Whether this node is for embedded platform views.
   bool IsPlatformViewNode() const;
 
   int32_t id = 0;
-  int32_t flags = 0;
+  SemanticsFlags flags{};
```

Comment:
Since you've initialized the default values of everything in SemanticsFlag struct you can just write this as `SemanticsFlags flags`